conf/idp.properties

```diff
@@ -23,10 +23,10 @@ idp.uw.mfa-optin-cookie = uw-optin-
 # users opting out of 2fa ( logical removal from above group )
 idp.uw.2fa-optout-urlbase = https://groups.uw.edu/group_sws/v3/group/u_weblogin_optout-2fa/effective_member/
 # users soon to be opted in to 2fa
-idp.uw.2fa-optin-soon-urlbase = https://groups.uw.edu/group_sws/v3/group/u_weblogin_optin-2fa-soon/effective_member/
-idp.uw.mfa-soon-cookie = uw-optin-soon-
+#idp.uw.2fa-optin-soon-urlbase = https://groups.uw.edu/group_sws/v3/group/u_weblogin_optin-2fa-soon/effective_member/
```

Comment:
I guess you could just delete these; github will have the history if anyone's interested.